Description: use embedded-io feature from postcard
 flavors::io::eio is gated behind embedded-io in v1.10
Author: Ananthu C V <weepingclown@disroot.org>
Forwarded: not-needed
Last-Update: 2024-11-20

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -83,3 +83,3 @@
 version = "1.0.4"
-features = ["use-std"]
+features = ["use-std", "embedded-io"]
 
